docs/src/snippets/figma-icon-color.mdx

```diff
@@ -0,0 +1,14 @@
+<p>
+  When you add a new icon to {props.component.match(/^[aeiou]/) ? "an" : "a"} {props.component} in
+  Figma, you need to adjust the color. When you change the icon, the icon color gets changed to the
+  default icon color. All Orbit icons have their defaults set to{" "}
+  <InlineToken value="paletteInkNormal" alternateName />.
+</p>
+
+After adding your icon,
+click the current color in the right sidebar (under **Selection Colors**)
+and select the color you want.
+
+<h3>Which color to choose for which {props.component}?</h3>
```

Comment:
I just remembered that you _might_ be able to use fragments for interpolation:
```suggestion
### Which color to choose for which <>{props.component}</>?
```
If so, the same can be done for all the other instances of where you had to convert to JSX just so you can interpolate, like the paragraph above, `<table>`, `<ul>` etc. (But not for situations like `<a>` where the value of `href` is a variable.)

Reply:
It doesn't work. Just displays `{props.component}` as part of the heading.

Reply:
Never mind then, I guess it's buggy.